specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -900,6 +900,191 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/inventoryPolicies/{blobInventoryPolicyName}": {
```

Comment:
Adding APIs to existing api-versions is no longer allowed.  See: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-changes-that-require-a-version-change

Reply:
I believe the point of having a preview version is that is not stable and customers must expect addition deletion etc changes.

I am keen to understand WHY I cannot add a brand new API in the PREVIEW version? What does it Break?
I did not get an answer to the WHY in the link you provided.

This PR is used for a private preview under the august2020-preview version. Not sure if you are expecting to release a new preview version for every addition removal of properties or an API.

Reply:
Additive (evolutionary) changes to preview versions seem to be allowed from that doc.

Public previews are released broadly but contain APIs that may change between previews and may be deleted prior to the final version. The X integer (indicating the revision of the preview) must be incremented for all breaking changes (resulting in a new API version). Evolutionary changes may be added as needed, as long as the change is communicated broadly.

Reply:
If this is intended for a private preview, it should not be targeted at the public repo

Reply:
Resolved offline. This is for public preview